integration/service/mail_test.go

```diff
@@ -0,0 +1,19 @@
+package service
+
+import (
+	"github.com/GenesisEducationKyiv/software-engineering-school-4-0-DimaL-cloud/internal/configs"
+	"github.com/GenesisEducationKyiv/software-engineering-school-4-0-DimaL-cloud/internal/service"
+	log "github.com/sirupsen/logrus"
+	"github.com/stretchr/testify/assert"
+	"testing"
+)
+
+func TestSendEmails_Success(t *testing.T) {
+	conf, err := configs.NewConfig("../../configs/config.yml")
```

Comment:
Agree, thx for advice

Reply:
Actually, to my mind, embed directive is not suitable in my case because we don't have access of files in parent directory (can't use '..') and we will need to duplicate config.yml and place it in the services folder. But if it is okay, let me know and I'll fix it